app/scripts/seller-transform.js

```diff
@@ -89,6 +89,39 @@ var sellerTransform = (function(_, relatedEntityTransform, commonTransforms) {
         return buckets;
     }
 
+    function processLocationGraph(records){
+        var data = [];
+        _.each(records, function(record){
+            var point = {}
+            point.date = record.key_as_string;
+
+            point.value = {}
+            if (record.localities.buckets){
+                sum = 0;
```

Comment:
The "sum" (line 100), "geoData" (line 102), and "city" (line 103) variables have all been created here as global variables.  Please use "var" to create them as local variables.